api/references/contribution-points.md

```diff
@@ -43,6 +44,23 @@ MetaDescription: To extend Visual Studio Code, your extension (plug-in) declares
 - [`viewsWelcome`](/api/references/contribution-points#contributes.viewsWelcome)
 - [`walkthroughs`](/api/references/contribution-points#contributes.walkthroughs)
 
+## contributes.authentication
+
+Contributes an authentication provider. This will set up an activation event for your provider and display it on in your extension's features.
+
+```json
+{
+  "contributes": {
+    "authentication": [
+      {
+        "label": "Azure Dev Ops",
+        "id": "azuredevops"
```

Comment:
Does the id match with the extension's backend implementation, or does it actually have to be specific to some set of IDs we have?

Reply:
it has to match what you register in your extension code.